.github/workflows/review-smoke-test.yml

```diff
@@ -24,6 +24,9 @@ jobs:
         - name: Install SFDX
           run: npm install -g  sfdx-cli
           
+        - name: Run NPM Install
+          run: npm install
```

Comment:
Can you make this as part of a seperate PR?

Reply:
I have done this already - not sure why it's showing in this PR exactly. I pulled into this branch from main, but there shouldn't be changes.